src/BookNavigator/search/a-search-result.js

```diff
@@ -35,17 +36,17 @@ export class BookSearchResult extends LitElement {
   }
 
   render() {
-    const { match } = this;
-    const { par = [] } = match;
-    const [resultDetails = {}] = par;
-    const pageNumber = Number.isInteger(resultDetails.page)
-      ? html`<p class="page-num">Page -${resultDetails.page}-</p>` : nothing;
+    /** @type {SearchInsideMatch} */
+    const match = this.match;
+    // TODO: Make this not use the global br instance
+    const pageIndex = window.br.leafNumToIndex(match.par[0].page);
```

Comment:
hi @cdrini thanks for this! can you pipe it thru via the `searchProvider`? that layer receives `br` instance, and we won't have to call global var..